src/label_studio_sdk/label_interface/control_tags.py

```diff
@@ -505,6 +505,13 @@ class SpanSelectionOffsets(SpanSelection):
 class ChoicesValue(BaseModel):
     choices: List[str]
 
+    # I don't know how Choices predictions with choice != 'multiple' was working without this...
```

Comment:
isn't it because of https://github.com/HumanSignal/label-studio-sdk/blob/master/src/label_studio_sdk/label_interface/control_tags.py#L419 ?

Reply:
Oh, good to know, will rewrite to use that existing function instead of a new one